Project/QtCreator/qctools-cli/qctools-cli.pro

```diff
@@ -1,5 +1,5 @@
-QT += core network
-QT -= gui
+QT += core gui multimedia network
```

Comment:
Can't we avoid to require GUI stuff for the CLI package?
It may create problems on servers without GUI support, either lack of QtGUI packages and/or company policy forbidding to install a X server..

Reply:
Hopefully one day - yes, although it will require some support from QtAVPlayer side. For now need to finish migration and get something working.

Reply:
@JeromeMartinez https://github.com/valbok/QtAVPlayer/issues/305

Reply:
Should QtGUI be avoided at all? QtAVPlayer requires to have QtGUI to use some features like `opengl`, 'egl', also to negotiate platform and choose right codec.

Reply:
It is a preference, in order to avoid to pull all GUI stuff on a server without GUI, but only if it is easy to do, I expected that all non GUI stuff related to GPU i.e. codec choice is not linked to QtGUI, but if it currently the case we can keep QtGUI for CLI up to someone motivated enough for sponsoring the removal of this binding if it is a big issue for the person.

Reply:
Thanks, removing QtGUI dependency at all.